teacher_class.rb

```diff
@@ -0,0 +1,11 @@
+require './person_class'
+class Teacher < Person
+  def initialize(specialization)
+    super()
+    @specialization = specialization
+  end
```

Comment:
- [ ] Here at the time of initialization we have to pass four parameters. (age, `specialization`, name = "Unknown", parent_permission = true). some of them have default values. Kindly use-
```ruby
 def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```